extensions/blocks/podcast-player/components/playlist.js

```diff
@@ -0,0 +1,93 @@
+/**
+ * External dependencies
+ */
+import classnames from 'classnames';
+import { memo } from '@wordpress/element';
+import { __ } from '@wordpress/i18n';
+
+/**
+ * Internal dependencies
+ */
+import { STATE_ERROR } from '../constants';
+
+const TrackError = memo( ( { link } ) => (
+	<div className="podcast-player__episode-error">
+		{ __( 'Episode unavailable', 'jetpack' ) }{ ' ' }
+		{ link && (
+			<span>
+				{ '(' }
+				<a href={ link } rel="noopener noreferrer nofollow" target="_blank">
+					{ __( 'Open in a new tab', 'jetpack' ) }
+				</a>
+				{ ')' }
+			</span>
+		) }
+	</div>
+) );
+
+const Track = memo( ( { track, isActive, isError, selectTrack, index } ) => {
+	return (
+		<li
+			className={ classnames( 'podcast-player__episode', {
+				'is-active': isActive,
+			} ) }
+		>
+			<a
+				className="podcast-player__episode-link"
+				href={ track.link }
+				role="button"
+				aria-pressed="false"
+				onClick={ e => {
+					// Prevent handling clicks if a modifier is in use.
+					if ( e.shiftKey || e.metaKey || e.altKey ) {
+						return;
+					}
+
+					// Prevent default behavior (opening a link).
+					e.preventDefault();
+
+					// Select track.
+					selectTrack( index );
+				} }
+				onKeyDown={ e => {
+					// Only handle the Space key.
+					if ( event.key !== ' ' ) {
```

Comment:
```suggestion
					if ( event.code !== ' Space' ) {
```

Minor: would it be more explicit to check against a named key code?

Reply:
I believe we've already talked about this during last iteration. `code` is unsupported in IE11 and that's why we kept `key`. 

Now we are in React, I think they do some sort of normalization but I don't have in mind how exactly it works.

Reply:
Seems like the `e.key === ' '` is the proper thing to use in React and other browsers are being normalized into this too. 
Issue I located: https://github.com/facebook/react/issues/5809

And I've found the relevant code from React that does the key normalization: https://github.com/facebook/react/blob/0140118e8e2766abaac2c833775434f83df81d56/packages/react-dom/src/events/getEventKey.js#L18

`e.code` is not being normalized so it will behave differently between browsers (like IE11 where it will not be available)